.github/workflows/test-build-system.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will thoroughly test the build system
+name: Complete build system test
```

Comment:
Seems a bit overkill but why not...

Reply:
Ported from another project, I guess it's there for any other targets added in future?